### ISSUE TITLE
Restore data very slowly please help

### ISSUE THREAD

**Issue by jinghuaz** [mhart/dynalite#77]
I now encounter a little trouble, I have from the awsdump table of data, now restored to the local dynalite very slow, I use the java program to write inside, is there any way to help me? Optimized at the speed of write recovery？Capacity of about 3T
Tks a lot.

**Comment by jinghuaz** [comment]
hi mhart， can you help me?~  thank you very much.

**Comment by mhart** [comment]
@jinghuaz are you saying you're trying to load 3TB of data into dynalite?

**Comment by jinghuaz** [comment]
yup，sure.because we want dump from AWS ,then restore to dynalite. For local index,so,i want to talk about you and hope you can help me do this.now dump is very fast,but restore very slow,3 or 4 k/s.another you can tell me your weixin or facebook?

**Comment by jinghuaz** [comment]
@mhart,I want to know how can quickly restore or write to dynalite. LevelDB?

**Comment by mhart** [comment]
@jinghuaz right, dynalite uses [levelup](https://github.com/Level/levelup), which can use LevelDB if you specify `--path` when you start it (by default it's just in-memory)

dynalite uses [sublevel](https://github.com/dominictarr/level-sublevel) to break the LevelDB into sub-tables for each of the DynamoDB tables and indexes. You can see how that structure works here:

https://github.com/mhart/dynalite/blob/master/db/index.js#L59-L81

**Comment by jinghuaz** [comment]
@mhart Do you have a webchat account?
